### PR TITLE
Feature/240 add desire icons

### LINKED ISSUE #197: Mood decline visualization
**Description**
Make an icon system for show mood status and declining/increasing

**Subtasks**
- [ ] Show current mood status
- [ ] Show mood declining
- [ ] Show mood increasing

### LINKED ISSUE #240: Add icons for desires
**Description**
Add icons above heads when vikings desire something

**Subtasks**
- [ ] Added icons

### PR DESCRIPTION
**Pull request by tzaarela**
Closes #240 
Closes #197

**Description**  
Changed from gameObject to sprite when vikings desire something.

**List of changes**  
- Sprite 
- Added to world canvas
- Added icons

### CODE REVIEW

File: Assets/BillboardLookAt.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BillboardLookAt : MonoBehaviour
```
Comment (MrJackey) [inline Assets/BillboardLookAt.cs:5]: This script could be good to keep but im not sure if using it on the visualisers are a good idea. Also the indentation, brace style and the file location is wonky. You should probably take a look at these points.

**Review by MrJackey — CHANGES_REQUESTED** [review]
Due to creating this branch in the middle of the bigger changes to models, a lot of things have broke. Maybe try and merge master into this.

File: Assets/BillboardLookAt.cs (hunk 1 above)
Comment (tzaarela) [reply Assets/BillboardLookAt.cs:5]: can you please explain why it´s not a good idea?

Comment (MrJackey) [reply Assets/BillboardLookAt.cs]: Currently progressbars are not billboard. The main issues are  that this results in the bubble sometimes covering the progressbar if the camera is above it. It also moves the speech bubble "indicatior" of who is talking away from the viking.

**Review by MrJackey — APPROVED** [review]
Think a smooth lerp between two colors would look better then an instant switch of color. Now it's just mixing billboard and not with progressbars that need to be solved

**Pull request merged**